### DISCUSSION TITLE
Get top 10 organizations with the GraphQL API

### DISCUSSION THREAD

**Discussion by toast-uz** [community/community#24454] [Apps, API and Webhooks]
<p>Hi,</p>
<p>Is it possible to get top 10 organizations which have a number of members, with the GraphQL API?</p>
<p>I’d like to analyze like this: </p>
<p><a href="https://resources.whitesourcesoftware.com/blog-whitesource/git-much-the-top-10-companies-contributing-to-open-source" rel="nofollow noopener">https://resources.whitesourcesoftware.com/blog-whitesource/git-much-the-top-10-companies-contributing-to-open-source</a></p>
<p>I was able to list 10 organizations by this query:</p>
<pre><code>query {
  search(first: 10, type: USER, query: "type:org"){
    userCount
    edges{
      node{
        ... on Organization {
          name
          membersWithRole{
            totalCount
          }
        }
      }
    }
  }
}
</code></pre>
<p>But I cannot find the right queryString for sorting by  the number of members.</p>
<p>Regarding repositories, there is an undocumented scalar field: “stars”. By using this, we can compare repositories by the number of stars. Is there some similar undocumented scalar fields on the organizaition?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There isn’t a method to <a href="https://help.github.com/en/articles/searching-users" rel="nofollow noopener">sort organizations</a> by the number of members. You could iterate through all of them and sort them yourself though.</p>
<p>I hope that helps!</p>
